src/libopensc/pkcs15-gemsafeV1.c

```diff
@@ -236,8 +237,9 @@ static int gemsafe_get_cert_len(sc_card_t *card)
 	 */
 	iptr = ibuf + GEMSAFE_READ_QUANTUM;
 	while ((size_t)(iptr - ibuf) < objlen) {
+		unsigned long flags = 0;
```

Comment:
this variable `flags` shadows the previous declaration of `flags`. I think just rest it to 0 here should be enough.

Reply:
Thanks for review, fixed.